W3ChampionsStatisticService/Hubs/WebsiteBackendHub.cs

```diff
@@ -16,17 +16,19 @@ public class WebsiteBackendHub(
     IW3CAuthenticationService authenticationService,
     ConnectionMapping connections,
     IHttpContextAccessor contextAccessor,
-    FriendRepository friendRepository,
     FriendRequestCache friendRequestCache,
-    IPersonalSettingsRepository personalSettingsRepository
+    IPersonalSettingsRepository personalSettingsRepository,
+    FriendCommandHandler friendCommandHandler,
+    FriendListCache friendListCache
```

Comment:
I don't think this needs the cache if it can just go through the handler, which uses a cache internally anyway

Reply:
Yes, thank you!